docs/_articles/en/release-information.md

```diff
@@ -5,6 +5,31 @@ lang: en
 
 ### To update the plugin, please follow the linked  [instructions](./en/getting-started/install/#upgrade-plug-in).
 
+## [v2.6.0](https://github.com/forcedotcom/sfdx-scanner/tree/v2.6.0) (02-17-2021)
+
+[Full Changelog](https://github.com/forcedotcom/sfdx-scanner/compare/v2.5.1...v2.6.0)
+
+### Release Summary
+
+* Support for SARIF JSON Format
+* Underlying PMD Engine is upgraded to 6.31.0 (from 6.30.0) 
+* RetireJS upgraded to v2.2.4
+
+**Closed issues:**
+
+- Generation of a code quality report in a JSON file that implements a subset of the Code Climate spec. [\#329](https://github.com/forcedotcom/sfdx-scanner/issues/329)
```

Comment:
We closed this issue without code change. Is it still worth putting in here?

Reply:
The closed issues are auto-generated. Although no code changes, the issues were closed with some instructions or comments from our side that helped resolve the issue. So, it might be a good idea to keep them. 

If both, you and @jbartolotta-sfdc feel we have to take it out, I can take both of them out. In the past, the closed issues with no code changes were included in this list.